lib/jekyll-default-layout/generator.rb

```diff
@@ -62,5 +65,9 @@ def post?(document)
     def page?(document)
       document.is_a?(Jekyll::Page)
     end
+
+    def index?(document)
+      document.url == "/"
```

Comment:
In the test fixture, there's an `index.html` file, which gets a url of `/` (and if they want `/index` to render, they're SOL here :smile:)